docs/running-workflows/launch-with-badge/index.md

```diff
@@ -0,0 +1,74 @@
+# Launch workflow using a badge
+
+To make it easier to launch a workflow, we provide a "Launch-on-REANA" badge that allows running a workflow with a single click.
+
+For example, to launch a [RooFit demo workflow](https://github.com/reanahub/reana-demo-root6-roofit) from GitHub on REANA instance at CERN, you can click on the badge below:
+
+[![Launch ROOT6 workflow from GitHub](https://www.reana.io/static/img/badges/launch-on-reana.svg)](https://reana.cern.ch/launch?url=https://github.com/reanahub/reana-demo-root6-roofit)
+
+*Note:* You will need to be logged in to REANA instance to be able to launch a workflow.
+
+To create your own badge, you will need to craft a URL that provides information to REANA instance on how to launch the workflow.
+The basic version of the URL looks like this:
+
+```text
+https://<REANA_INSTANCE_DOMAIN>/launch?url=<WORKFLOW_SOURCE_URL>
+```
+
+Where:
+
+- `REANA_INSTANCE_DOMAIN` is a domain of the REANA instance that will be used to launch the workflow;
+- `url` parameter, where `WORKFLOW_SOURCE_URL` is a link to workflow that needs to be launched.
+
+For example, the above badge for RooFit demo workflow is using this URL:
+
+```text
+https://reana.cern.ch/launch?url=https://github.com/reanahub/reana-demo-root6-roofit
+```
+
+Moreover, you can specify additional parameters that can modify the workflow launch.
+They are described in the ["Additional parameters"](#additional-parameters) section below.
+
+## Supported workflow sources
+
+Below you can find a list of supported workflow sources that can be used in ``url`` parameter.
+
+### GitHub
+
+You can use GitHub repositories to launch a workflow. Below are examples of supported URL formats:
+
+- `https://github.com/reanahub/reana-demo-root6-roofit`
+- `https://github.com/reanahub/reana-demo-root6-roofit.git`
+- `https://github.com/reanahub/reana-demo-root6-roofit/tree/master`
+
+### ZIP
+
+You can provide a link to a ZIP file to launch a workflow. Below are examples of supported URL formats:
+
+- `https://zenodo.org/record/5752285/files/circular-health-data-processing-master.zip`
+
+## Additional parameters
+
+### name
+
+By default, REANA will try to determine the name of the workflow from the URL.
+But, in some cases, you may want to specify the name of the workflow.
+You can do this by adding a `name` parameter to the URL:
+
+```text
+https://reana.cern.ch/launch?url=https://github.com/reanahub/reana-demo-root6-roofit?name=my-new-name
+```
+
+## Add badge in Markdown
```

Comment:
Possible text for badges:

## Launcher badges

In order to facilitate launching your analysis workflows directly from GitHub or GitLab, you may want to generate a badge pointing to the launcher URL with the desired arguments and then include this badge in your analysis documentation.

The standard badge looks like:

   Launch-on-REANA

and its source code in Markdown is:

  FIXME

It is sufficient to generate your launch URL with your desired arguments to generate one or more custom badges to include in your documentation.

For example, here is a badge for RooFit demo workflow with name "myworkflow" and generating once 10000 events, once 20000 events:

  FIXME-badge-1
  FIXME-badge-2

Note how the badges use custom non-standard text.

Reply:
BTW one more thing: you can show not only Markdown source, but also HTML source, for a badge. So that people could add badges easily to CERN Open Data, CERN Analysis Preservation, Zenodo or any other system for our generic URL fetcher use case. (Not everybody uses Markdown.)